internal/domain/filter.go

```diff
@@ -254,6 +256,63 @@ func (f *Filter) Validate() error {
 		return fmt.Errorf("error validating filter size limits: %w", err)
 	}
 
+	for _, external := range f.External {
+		if external.Type == ExternalFilterTypeExec {
+			if external.ExecCmd != "" {
```

Comment:
```suggestion
			if external.ExecCmd != "" && external.Enabled {
```
Would this make more sense? Right now it errors on not found, even if disabled.

Reply:
Yeah good idea, that way a user can still save it if disabled and fix it :+1: